seismostats/utils/binning.py

```diff
@@ -87,6 +89,11 @@ def binning_test(
             (with bin-sizes delta_x)
         delta_x:    size of the bin
         tolerance:  tolerance for the comparison
+        false_if_larger: if True (default), the function will return False if a
```

Comment:
this is quite confusing, maybe easier if it's called true_if_larger and the default is false? setting false_if_larger to false means... true if larger? (reminds me of some of my own confusing code)

Reply:
Agree that it might be a bit confusing.. But true_if_larger does not really capture the meaning - it should also give back true when the smaller binning is correct, but the larger one isn't. So, false_if_larger is actually better: if it's set to False, the function does not give back False in the case when a larger binning is also correct.

Not sure, do you have another suggestion? I thought about check_larger_binning (but then it might not make sense to give back a warning.. )